share/jupyterhub/static/less/page.less

```diff
@@ -44,6 +44,12 @@
   padding: 8px;
 }
 
+/* #thenavbar{
+  ul li a {
+    color: #222;
+  }
+}*/
```

Comment:
empty css, commented-out? Maybe there's a bootstrap LESS variable we should be setting to control the link color, rather than these specific navbar elements.

Reply:
**File:**....share/jupyterhub/static/components/bootstrap/less/variables.less**

Changing the `@navbar-default-link-color:` and `@navbar-default-link-hover-color: ` in variables.less fixed this.

**Report:**
<img width="1280" alt="Screenshot 2022-12-16 at 17 42 33" src="https://user-images.githubusercontent.com/84882370/208158253-16000501-ce64-4cc1-a605-d59a3bbc81e4.png">

Reply:
Good find! Looks like this hasn't been added yet, though. Still working on a commit?

Reply:
Yes, I am. On running `git status`, I get the prompt `Your branch is up to date with origin/main`.

I am inferring this is because share/jupyterhub/static/components is in the .gitignore file. Please what can I do?

Reply:
Bootstrap variables should be overridden in `share/jupyterhub/static/less/variables.less`. Files in components should not be edited.

Reply:
> Bootstrap variables should be overridden in `share/jupyterhub/static/less/variables.less`. Files in components should not be edited.

Okay. Thanks